pkg/registry/networking/networkpolicy/strategy.go

```diff
@@ -86,3 +91,25 @@ func (networkPolicyStrategy) ValidateUpdate(ctx context.Context, obj, old runtim
 func (networkPolicyStrategy) AllowUnconditionalUpdate() bool {
 	return true
 }
+
+// Drops Network Policy Disabled fields if Feature Gate is also disabled.
+// This should be used in future Network Policy evolutions
+func dropNetworkPolicyDisabledFields(newNetPol *networking.NetworkPolicy, oldNetPol *networking.NetworkPolicy) {
```

Comment:
Can we add a unit test for this?

Reply:
Yeap, I was waiting the review before adding all the unit tests, to know if those were in the right place :)